mkdocs.yml

```diff
@@ -42,3 +41,18 @@ nav:
             - governance/CONTRIBUTING.md
             - editing/schema.md
             - editing/extensions.md
+plugins:
+    - search
+    - print-site:
+        add_table_of_contents: false
+        enumerate_headings_depth: 3
+        exclude:
```

Comment:
Now that the validator is merged, it should be excluded too `validator/index.md`

Reply:
Done.